app/database/server/models/group_membership.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {Relation} from '@nozbe/watermelondb';
+import Model, {Associations} from '@nozbe/watermelondb/Model';
+import {immutableRelation} from '@nozbe/watermelondb/decorators';
+
+import {MM_TABLES} from '@constants/database';
+import Group from '@typings/database/group';
+import User from '@typings/database/user';
+
+const {GROUP, GROUP_MEMBERSHIP, USER} = MM_TABLES.SERVER;
+
+/**
+ * The GroupMembership model represents the 'association table' where many groups have users and many users are in
+ * groups (relationship type N:N)
+ */
+export default class GroupMembership extends Model {
```

Comment:
add lazy queries to get all users & all groups

Reply:
done